static/app/app/list/templates/lunch-move.html

```diff
@@ -17,6 +17,12 @@
                 </span>
             </div>
         </div>
+        <% if (isOwnMove(move)) { %>
+            <div class="delete">
+                <span>Delete</span>
+                <span class="glyphicon glyphicon-trash"></span>
```

Comment:
Adding this in as the full word 'Delete' and having it take up the width of a lunchmove is pretty heavy from a design perspective. I think it would be better if it was an x icon in the upper right of the lunchmove. As we do lose some clarity with making it an x, it might good to add a [boostrap tooltip](http://getbootstrap.com/javascript/#tooltips) to it which says "Delete?" on hover.